src/style.css

```diff
@@ -153,13 +258,88 @@ footer {
   width: 100%;
 }
 
-/* Comments-Popup CSS */
-.moviediv {
-  display: grid;
-  grid-template-columns: 33% 33% 33%;
-  grid-template-rows: auto auto auto auto;
-  gap: 10px;
-  padding-left: 10%;
-  padding-right: 10%;
-  margin-bottom: 50px;
+@media screen and (max-width: 768px) {
+  header {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    margin-bottom: 20px;
+    column-gap: 10px;
+  }
+
+  header img {
+    width: 150px;
+    height: 60px;
+    animation-name: showlogo;
+    animation-duration: 4s;
+  }
+
+  @keyframes showlogo {
+    from {
+      width: 1px;
+      height: 1px;
+    }
+
+    to {
+      width: 150px;
+      height: 60px;
+    }
+  }
+
+  #categories {
+    column-gap: 10px;
+  }
+
+  #categories li {
+    list-style: none;
+    font-size: 16px;
+    font-weight: bold;
+  }
+
+  .title {
+    font-size: 14px;
+  }
+
+  .like {
+    width: 20px;
+    height: 20px;
+  }
+
+  .like-text {
+    font-size: 12px;
+    font-weight: 200;
+  }
+
+  .comments {
+    font-size: 14px;
+  }
+
+  .reservations {
+    font-size: 14px;
+  }
+
+  .moviediv {
+    display: grid;
+    grid-template-columns: 100%;
+    grid-template-rows: auto;
+    gap: 10px;
+    padding-left: 10%;
+    padding-right: 10%;
+    margin-bottom: 50px;
+  }
+
+  .modal {
+    display: none;
+    position: fixed;
+    background-color: rgb(206, 209, 202);
+    width: 100%;
+    height: 100%;
+    bottom: 0;
+    left: -160px;
+    overflow-y: scroll;
+  }
```

Comment:
Try to give a popup left: 0 and the popup fill the inter-space

Reply:
Thank you. I resolved this issue. You can check now.